### ISSUE TITLE
Images cannot be resized within a Table

### ISSUE THREAD

**Issue by tabedzki** [MSzturc/obsidian-advanced-slides#141]
**Describe the bug**
A clear and concise description of what the bug is. I believe this arises from the table reading the `|` symbol which is used to denote the image wiidth. I've tried versions where the `|` is in a `! [[ ]] ` format as well as the `![ ]( )` syntax. Both do not yield what the Obsidian preview yields.

**Samples to Reproduce**
Paste the following into a markdown example and try to run it.

Example Table: 
```markdown

Example
 
| Bottlebrush | Parameters                                                                                                        | Image | 
| ----------- | ----------------------------------------------------------------------------------------------------------------- | ----- | 
| ![\| 100](https://upload.wikimedia.org/wikipedia/en/thumb/8/80/Wikipedia-logo-v2.svg/1200px-Wikipedia-logo-v2.svg.png) | test  | text |
| test        | ![\|10](https://upload.wikimedia.org/wikipedia/en/thumb/8/80/Wikipedia-logo-v2.svg/1200px-Wikipedia-logo-v2.svg.png)       |       | 
| test        | ![\|100](https://upload.wikimedia.org/wikipedia/en/thumb/8/80/Wikipedia-logo-v2.svg/1200px-Wikipedia-logo-v2.svg.png)       | ![\|100](Pasted_image_20220804220818.png)      | 


```

**Expected behavior**
To have the table either resize the content natively to have everything fit on the slide or to use the requested pixel size to resize the images.

**Comment by tabedzki** [comment]
Semi-related, there a way to have the table or content automatically fit? I don't know CSS at all and was hoping for something with Obsidian that would work out of the box for content rescaling.

**Comment by fnumatic** [comment]
Resizing images inside table cells is not that easy.  By default `<td> </td>` should grow with the size of the content.
Regarding your example:
```
## Example

| Bottlebrush | Parameters                        | Image |
| ----------- | --------------------------------- | ----- |
| test        | ![](https://tinyurl.com/44p5tkwr) | test  |
| test        | ![](https://tinyurl.com/44p5tkwr) |       |
| test        | ![](https://tinyurl.com/44p5tkwr) |       |
```

This styling can somewhat help.
```css
<style>
.reveal td{
  position: relative;
}
.reveal td img{
  position: absolute;
  top: 0;
}
</style>
```
![screenshot css img table](https://user-images.githubusercontent.com/1319530/197271189-8b2cf498-b899-41a9-904d-354d336657c8.png)

**Comment by MSzturc** [comment]
Yeah, by default revealjs doesn't optimize layouting of tables, currently advanced slides doesn't modify the rendering of table. This will be a big change. I find it a pity that markdown spec has no syntax to describe table column & row properties. I will have a look at this topic when things get a little quieter in my daily job.

**Comment by tabedzki** [comment]
Ahh ok thanks; this isn't a high priority request from my end since I figured out a solution for the presentation I wanted to do.
I'm surprised that the Markdown standard itself doesn't implement something like this though.
